src/components/PeopleFilters.tsx

```diff
@@ -27,66 +67,50 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
+            {CENTURIES_LINKS.map(century => (
+              <SearchLink
+                data-cy="century"
+                key={century}
+                className={classNames('button mr-1', {
+                  'is-info': centuries.includes(century),
+                })}
+                params={{
+                  centuries: centuries.includes(century)
+                    ? centuries.filter(cent => century !== cent)
+                    : [...centuries, century],
+                }}
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable